recipes/tox/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "tox" %}
+{% set version = "2.7.0" %}
+{% set sha256 = "9c3bdc06fe411d24015e8bbbab9d03dc5243a970154496aac13f9283682435f9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - py >=1.4.17 
+    - pluggy >=0.3.0,<1.0
+    - virtualenv >=1.11.2
+    - setuptools
+   
+test:
+  imports:
+    - tox
+
+about:
+  home: https://github.com/tox-dev/tox
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Tox is a generic virtualenv management and test command line tool you can use for
+    - checking your package installs correctly with different Python versions and interpreters
+    - running your tests in each of the environments, configuring your test tool of choice
+    - acting as a frontend to Continuous Integration servers, greatly reducing boilerplate
+      and merging CI and shell-based testing.'
```

Comment:
Should be "virtualenv-based automation of test activities". Feel free to include `[ci skip]` in the message of the commit with this change.

ref: https://github.com/tox-dev/tox/blob/2.7.0/setup.py#L84